docs/crocks/Either.md

```diff
@@ -0,0 +1,89 @@
+# Either
+
+`Either a b`
+
+Crock which --
+
+The "`Either`" crock creates disjunction with a linear data flow.
+
+```js
+--
+```
+
+`Either` exposes these constructors and instances:
```

Comment:
Sorry one last thing I missed on the last review, would it be possible to change this to something like:

`Either` exposes the following constructor and instance functions: